price-feeder/oracle/provider/websocket_controller.go

```diff
@@ -134,6 +134,7 @@ func (wsc *WebsocketController) subscribe(msgs []interface{}) error {
 		if err := wsc.SendJSON(jsonMessage); err != nil {
 			return fmt.Errorf(types.ErrWebsocketSend.Error(), wsc.providerName, err)
 		}
+		time.Sleep(1 * time.Second) // avoid hitting provider send limit
```

Comment:
I think we can get rid of this sleep now for when the multisocket solution is merged in. This subscribe method will only subscribe to one pair rather than looping through multiple.